IntelliTect.TestTools.TestFramework/TestCase.cs

```diff
@@ -59,6 +60,16 @@ public TestCase(string testCaseName, string testMethodName, int testCaseId, ISer
         /// <exception cref="TestCaseException">The exception describing a test failure.</exception>
         /// <exception cref="AggregateException">Occurs when finally blocks fail, or the test fails and at least one finally block fails.</exception>
         public void Execute()
+        {
+            ExecuteAsync().ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
```suggestion
            Task.Run(() => ExecuteAsync()).GetResult();
```

For details see: https://devblogs.microsoft.com/dotnet/should-i-expose-synchronous-wrappers-for-asynchronous-methods/